tests/integration/services/test_ruleset_activate.py

```diff
@@ -144,24 +145,10 @@ def get_activation_stats():
     }
 
 
-@pytest.mark.django_db
-@mock.patch("subprocess.run")
-def test_rulesets_activate_with_errors(run_mock: mock.Mock, init_data):
-    assert models.ActivationInstance.objects.count() == 0
-
-    ActivateRulesets().activate(
-        activation=init_data.activation, deployment_type="bad_type"
-    )
-
-    assert (
-        models.ActivationInstance.objects.first().status
-        == ActivationStatus.FAILED.value
-    )
-
-
```

Comment:
Would this still be useful to have as a test using the default the restart policy?

Reply:
`subprocess.run` is used by the deployment type of `local`, which had been removed by the previous refactor